.github/workflows/release.yml

```diff
@@ -21,13 +21,19 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.19
+          go-version: 1.21
       - if: startsWith(github.ref, 'refs/tags/a/v')
         shell: bash
         run: echo "flags=-f ./a/.goreleaser.yaml" >> $GITHUB_ENV
       - if: startsWith(github.ref, 'refs/tags/b/v')
         shell: bash
         run: echo "flags=-f ./b/.goreleaser.yaml" >> $GITHUB_ENV
+        if: startsWith(github.ref, 'refs/tags/internal/v')
+        shell: bash
+        run: echo "flags=-f ./internal/.goreleaser.yaml" >> $GITHUB_ENV
+        if: startsWith(github.ref, 'refs/tags/sacct/v')
```

Comment:
though this has a logic setup here for sacct tags, the workflow is not triggered yet on tags with sacct, therefore it'll never make it to this logic. Update the on push section to also reflect the needed tags

Reply:
fixed